### ISSUE TITLE
Can we have resolution switching for mobile devices 

### ISSUE THREAD

**Issue by bijfy** [martinbjeldbak/acestream-http-proxy#35]
Just wondering if it is possible to add resolution pick to lower eg 360 or 480 px to save some mobile data. ?

**Comment by iercan** [comment]
I don't think so. As far as I know, the engine doesn't determine stream quality. It just download and share it as it is.
